src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -1020,6 +1022,55 @@ registerVisualizationLayerDefinition({
   },
 });
 
+registerVisualizationLayerDefinition({
+  identifier: "fontra.status.color",
+  name: "Status color",
+  selectionMode: "all",
+  userSwitchable: true,
+  defaultOn: false,
+  zIndex: 100,
+  screenParameters: {
+    minThickness: 3,
+    maxThickness: 15,
+  },
+  draw: (context, positionedGlyph, parameters, model, controller) => {
+    const statusFieldDefinitions =
+      model.fontController.customData["fontra.sourceStatusFieldDefinitions"];
+    if (!statusFieldDefinitions) {
+      return;
+    }
+
+    const sourceIndex = positionedGlyph.glyph.sourceIndex;
+    if (sourceIndex === undefined) {
+      return;
+    }
+
+    const status =
+      positionedGlyph.varGlyph.sources[sourceIndex].customData[
+        "fontra.development.status"
+      ];
+    if (status === undefined) {
+      return;
+    }
+
+    const color = [...statusFieldDefinitions[status].color];
+    if (positionedGlyph.isEditing) {
+      // in editing mode make opacity 50%
+      color[3] = color[3] * 0.5;
+    }
+
+    const thickness = clamp(
+      0.05 * model.fontController.unitsPerEm,
+      parameters.minThickness,
+      parameters.maxThickness
+    );
+    context.fillStyle = rgbaToCSS(color);
+    context.beginPath();
```

Comment:
There is `context.fillRect()` as a shortcut for these three lines.

Reply:
Thank you. I changed it. Please see #1346